app/displayers.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
+from .models import Book
+
+
+class BookDisplayer(ABC):
```

Comment:
I don't quite understand why you named your class `Displayer`. Could you explain the reasoning behind that choice?